tools/targetbrowser/backup.go

```diff
@@ -25,8 +31,13 @@ func (c *Client) GetBackups(options *BackupListOptions) error {
 		return err
 	}
 	queryParam := values.Encode()
-	fmt.Printf("\n Backup URL: %s\n", fmt.Sprintf("%s/backup?%s", c.baseURL, queryParam))
-	req, err := http.NewRequest("GET", fmt.Sprintf("%s/backup?%s", c.baseURL, queryParam), nil)
+	return c.TriggerAPI(backupEndPoint, queryParam)
+
+}
+
+func (c *Client) TriggerAPI(apiEndPoint, queryParam string) error {
+	log.Debugf("Base URL of target-Browser: %s, API endPoint is %s and Query Param is %s.", c.baseURL, apiEndPoint, queryParam)
```

Comment:
not needed.. Remove all the log statements

Reply:
Removed all logs